cmd/root.go

```diff
@@ -168,7 +168,12 @@ func getCacheDir(imageName string) (string, error) {
 	}
 	rootDir := filepath.Join(cacheDir, ".container-diff", "cache")
 	imageName = strings.Replace(imageName, string(os.PathSeparator), "", -1)
-	return filepath.Join(rootDir, filepath.Clean(imageName)), nil
+	return filepath.Join(rootDir, cleanFilePath(imageName)), nil
+}
+
+func cleanFilePath(filePath string) string {
```

Comment:
this would be better suited in `pkg/util/fs_utils.go` or somewhere similar. would you mind making this a public function somewhere over there, and then making the list of characters in the replacer a `const []string`?

Reply:
Sounds good.